rbac/management/tasks.py

```diff
@@ -18,10 +18,17 @@
 from __future__ import absolute_import, unicode_literals
 
 from celery import shared_task
+from django.core.management import call_command
 from management.principal.cleaner import clean_tenants_principals
 
 
 @shared_task
 def principal_cleanup():
     """Celery task to clean up principals no longer existing."""
     clean_tenants_principals()
+
+
+@shared_task
+def run_migrations_in_worker():
+    """Celery task to run migrations."""
+    call_command("migrate_schemas")
```

Comment:
Do we need any error checking/handling on this, or will Django handle, say, a failed migration for us?

Reply:
Not really, because it kicks it off in a background process. However, if a migration fails, it should continue with the next, but that's something that would be baked into django-tenant-schemas I believe. We will get the logs regardless of what happens, in our worker pod though. So really I think this would be the same as running them in a process the way we do now, without our own error handling, but rather letting the library do that. Open to suggestions if you think we should try to do more, though!